gobblin-service/src/main/java/org/apache/gobblin/service/modules/utils/FlowCompilationValidationHelper.java

```diff
@@ -95,6 +95,7 @@ public Optional<Dag<JobExecutionPlan>> createExecutionPlanIfValid(FlowSpec flowS
     }
 
     addFlowExecutionIdIfAbsent(flowMetadata, optionalFlowExecutionId, jobExecutionPlanDagOptional.get());
+    // TODO: emit flow compilation metric
```

Comment:
the line below already emits a metric, unless you mean like a count?

Reply:
this was also left in by mistake, updated